src/installer/managed/Microsoft.NET.HostModel/MachO/EmbeddedSignatureHeader.cs

```diff
@@ -0,0 +1,33 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.InteropServices;
+
+namespace Microsoft.NET.HostModel.MachO;
+
+[StructLayout(LayoutKind.Sequential)]
```

Comment:
nit: maybe remove all `StructLayout`? It's default layout, there is no Pack attribute specified or no P/Invoke?
```suggestion
namespace Microsoft.NET.HostModel.MachO;

```

Reply:
Removing the `StructLayout` causes 'unused field' warnings. I think the StructLayout makes more sense rather than suppressions and makes it clear that these are representations of unmanaged data, but am fine to change it if there's a better solution.